4001-5000/LIT4145Onausp.xml

```diff
@@ -4,7 +4,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">On auspicious and inauspicious days</title>
+            <title xml:id="t1" xml:lang="gez">ሐሳበ፡ ጽንዕት፡ ወጽንጽንት፡</title>
+            <title corresp="#t1" xml:lang="gez">Ḥasāba ṣǝnʿǝt wa-ṣǝnṣǝnǝt</title>
```

Comment:
Ḥassāba ṣǝnǝʿt wa-ṣǝnṣǝnt, no? They are feminine participles, as I understand it